minicluster/src/main/java/org/apache/accumulo/minicluster/MiniAccumuloConfig.java

```diff
@@ -108,6 +108,19 @@ public MiniAccumuloConfig setZooKeeperStartupTime(long zooKeeperStartupTime) {
   }
 
   /**
+   * Configure an existing ZooKeeper instance to use. Calling this method is optional. If not set, a new ZooKeeper instance is created.
+   *
+   * @param existingZooKeepers
+   *          Connection string for a already-running ZooKeeper instance
+   *
+   * @since 1.7.0
+   */
+  public MiniAccumuloConfig setExistingZooKeepers(String existingZooKeepers) {
```

Comment:
> I'm used to seeing a note on Javadoc which states when a setter is overloaded. Personally, that's sufficient.

Since its really easy to do, I think structuring the API to fail fast would be better.  It avoids one case of strange failures deep in MAC that are possibly hard to debug.